msteams-platform/concepts/device-capabilities/location-capability.md

```diff
@@ -10,54 +10,58 @@ ms.author: surbhigupta
+. The update for app manifest is as follows:
 
 ``` json
 "devicePermissions": [
     "geolocation",
 ],
 ```
 
-> [!NOTE]
-> * The **Request Permissions** prompt is automatically displayed when a relevant Teams API is initiated. For more information, see [request device permissions](native-device-permissions.md).
-> * Device permissions are different in the browser. For more information, see [browser device permissions](browser-device-permissions.md).
+> **Note** </br>
+• The request permissions prompt is automatically displayed when a relevant Teams API is initiated. For more information, see [request device permissions](native-device-permissions.md).</br>
+• The device permissions are different in the browser. For more information, see [browser device permissions](browser-device-permissions.md).
+
 
 ## Location APIs
 
 You must use the following set of APIs to enable your device's location capabilities:
 
-| API      | Description   |
-| --- | --- |
-|[getLocation](/javascript/api/@microsoft/teams-js/microsoftteams.location?view=msteams-client-js-latest#getLocation_LocationProps___error__SdkError__location__Location_____void_&preserve-view=true) | Gives user’s current device location or opens native location picker and returns the location chosen by the user. |
-|[showLocation](/javascript/api/@microsoft/teams-js/microsoftteams.location?view=msteams-client-js-latest#showLocation_Location___error__SdkError__status__boolean_____void_&preserve-view=true) | Shows location on map. |
+| API      | Description |Input configuration `allowChooseLocation` |Input configuration `showMap` |
+| --- | --- |--- |--- |
+|`getLocation`| Gives user’s current device location or opens native location picker and returns the location chosen by the user. |True: Users can choose any location of their choice.</br> False: users cannot change their current location |False: The current location is fetched without displaying the map. | 
+|`showLocation`| Shows location on map. |True: Users can choose any location of their choice.</br> False: users cannot change their current location| False: The current location is fetched without displaying the map. | 
 
-> [!NOTE]
-> The `getLocation()` API comes along with following [input configurations](/javascript/api/@microsoft/teams-js/locationprops?view=msteams-client-js-latest&preserve-view=true), `allowChooseLocation` and `showMap`. <br/> If the value of `allowChooseLocation` is *true*, then the users can choose any location of their choice.<br/>  If the value is *false*, then the users cannot change their current location.<br/> If the value of `showMap` is *false*, the current location is fetched without displaying the map. `showMap` is ignored if `allowChooseLocation` is set to *true*.
+> **Note**</br>
+• Desktop doesn't support capturing current location.
+• If `allowChooseLocation` is set to *true*, the `showMap` is ignored.
```

Comment:
Add this info in the table: If `allowChooseLocation` is set to *true*, the `showMap` is ignored.
Please confirm the genuinity of the statement: Desktop doesn't support capturing current location.